app/templates/suppliers/details.html

```diff
@@ -74,10 +70,74 @@
     {{ summary.field_name('Supplier summary') }}
     {{ summary.text(supplier.description) }}
   {% endcall %}
-  {% call summary.row() %}
-    {{ summary.field_name('Clients') }}
-    {{ summary.list(supplier.clients) }}
-  {% endcall %}
 {% endcall %}
 
+{{ summary.heading("Registration information", id="registration_information") }}
+{{ summary.description("This is used to process your applications and manage your participation on frameworks.") }}
+{% call(item) summary.mapping_table(
+  caption='Registration information',
+  field_headings=[
+    'Label',
+    'Value'
+  ],
+  field_headings_visible=False
+) %}
+
+  {% if supplier.registeredName %}
+    {% call summary.row() %}
+      {{ summary.field_name('Registered company name') }}
+      {{ summary.text(supplier.registeredName) }}
+    {% endcall %}
+  {% endif %}
+
+  {% if supplier.companiesHouseNumber %}
+    {% call summary.row() %}
+      {{ summary.field_name('Companies House number') }}
+      {{ summary.text(supplier.companiesHouseNumber) }}
+    {% endcall %}
+  {% endif %}
+
+  {% if supplier.registrationCountry and not supplier.companiesHouseNumber %}
+    {% call summary.row() %}
+      {{ summary.field_name('Where your business was established') }}
+      {{ summary.text(supplier.registrationCountry|upper) }}
+    {% endcall %}
+  {% endif %}
```

Comment:
Are we *really* sure we want to show this when we have such incomplete data? (and when we do have data it will presumably come through as its ISO3166 code, leaving some possibly confused users)

Reply:
I don't know... it will only exist for UK-based suppliers who have filled out a recent declaration.  Seeing "GB" shouldn't be that confusing to them?

Reply:
Everyone else will see nothing and be none the wiser.

Reply:
Until someone goes and writes a script to populate it, not realizing that it will just show the ISO code raw. Whatever, it doesn't really matter - most people recognize their country's ISO3166-1a2 code anyway